src/cluster.c

```diff
@@ -4735,9 +4870,11 @@ void clusterGenNodesSlotsInfo(int filter) {
 }
 
 void clusterFreeNodesSlotsInfo(clusterNode *n) {
-    zfree(n->slot_info_pairs);
-    n->slot_info_pairs = NULL;
-    n->slot_info_pairs_count = 0;
+  if (n->slot_info_pairs != NULL) {
```

Comment:
This should be necessarily, zfree works on NULL and we're just zeroing information.

Reply:
Got it. Old habit.